controllers/storagecluster/storagecluster_controller.go

```diff
@@ -91,6 +91,10 @@ type StorageClusterReconciler struct {
 	IsNoobaaStandalone bool
 }
 
+func skipNoobaaCrdWatch() bool {
+	return os.Getenv("SKIP_NOOBAA_CRD_WATCH") == "true"
```

Comment:
where is this `SKIP_NOOBAA_CRD_WATCH`  coming from?

Reply:
Can it be more generic instead of being specific to Nooba?

Reply:
It'd need to be a list of CRDs to allow that. Any generic name would just cause confusion.
We could have another variable for OB/OBC?

Reply:
+1 to @umangachapagain 

This variable will be coming from the sub. Ms will add this variable in the sub and OLM will inject the variable into the operators.

Reply:
ok. @umangachapagain Will that be a different PR for OB/OBC?

Reply:
It depends on what we decide on. Mainly it depends on rook how it wants to proceed with the fix.